src/main/kotlin/org/rust/ide/annotator/RsSyntaxErrorsAnnotator.kt

```diff
@@ -297,17 +298,26 @@ private enum class TypeArgumentKind(private val elementClass: KClass<*>, val arg
     }
 }
 
-private fun require(el: PsiElement?, holder: AnnotationHolder, message: String, vararg highlightElements: PsiElement?): Annotation? =
+private fun require(el: PsiElement?, holder: AnnotationHolder, message: String, vararg highlightElements: PsiElement?): Unit? =
     if (el != null) null
-    else holder.createErrorAnnotation(highlightElements.combinedRange ?: TextRange.EMPTY_RANGE, message)
+    else {
+        holder.newAnnotation(HighlightSeverity.ERROR, message)
+            .range(highlightElements.combinedRange ?: TextRange.EMPTY_RANGE).create()
```

Comment:
I'm not sure how `TextRange.EMPTY_RANGE` worked with the old API, maybe such annotations was ignored, but with new API I sure it's an error. Let's not create the annotation if `highlightElements.combinedRange` is null